axolotl-web/src/pages/Register.vue

```diff
@@ -61,6 +61,10 @@ export default {
     var userLang = navigator.language || navigator.userLanguage;
     this.$language.current = userLang;
     document.getElementById("VuePhoneNumberInput_phone_number").focus();
+    let loader = document.getElementById('initial-loader');
```

Comment:
@nanu-c Is there a way to make a function global so I don't duplicate it everywhere?

Reply:
make a component and include it in the header `axolotl-web/src/components/Header.vue`  and then it should work.